client/components/ReviewRating/ReviewList/Helpfulness.jsx

```diff
@@ -1,13 +1,24 @@
-import React from 'react';
 import PropTypes from 'prop-types';
+import React, { useState } from 'react';
 import axios from 'axios';
 
 const Helpfulness = ({ review, fetchReviews }) => {
+  const [clickHelpful, setClickHelpful] = useState(false);
```

Comment:
When naming these things, I'd try to use variable names that describe it's present state. Something like `[hasClickedHelpful, setHasClickedHelpful]`